doc/source/user/tutorial-svd.rst

```diff
@@ -303,92 +237,86 @@ As expected, you should see something like
     >>> linalg.norm(img_gray - U @ Sigma @ Vt)
     1.3926466851808837e-12
 
-(The actual result of this operation might be different depending on your
-architecture and linear algebra setup. Regardless, you should see a small
-number.)
+(Your result may be different depending on your architecture and linear
+algebra setup. Regardless, you should see a small number.)
 
-We could also have used the `numpy.allclose` function to make sure the
+We can also use the `numpy.allclose` function to make sure the
 reconstructed product is, in fact, *close* to our original matrix (the
-difference between the two arrays is small)::
+difference of the two arrays is small)::
 
     >>> np.allclose(img_gray, U @ Sigma @ Vt)
     True
 
-To see if an approximation is reasonable, we can check the values in ``s``::
+Now recall that :math:`\Sigma` orders the singular values starting with
+the most important. Can we find "unimportant" values to eliminate?
+
+Let's plot ``s``::
 
     >>> plt.plot(s)
 
 .. plot:: user/plot_gray_svd.py
     :align: center
     :include-source: 0
-    
-In the graph, we can see that although we have 768 singular values in
-``s``, most of those (after the 150th entry or so) are pretty small. So it
-might make sense to use only the information related to the first (say, 50)
-*singular values* to build a more economical approximation to our image.
 
-The idea is to consider all but the first ``k`` singular values in
-``Sigma`` (which are the same as in ``s``) as zeros, keeping
-``U`` and ``Vt`` intact, and computing the product of these matrices
-as the approximation.
+Although ``s`` has 768 singular values, most of them are pretty small. To
+build a more economical approximation, it might make sense to use only the
+first 50 or so.
```

Comment:
In both the original text an here, there seems to be a disconnect between the number of singular values to keep in the text (50) and the number used in the code example (10). Either is fine but I think it would be an improvement to have the text and the code sync up.

Reply:
This isn't a bug, it's a feature. 50 seems like a reasonable bound from the graph;  and for fun the tutorial says let's set k to 10, a really low value, to see how much of the image we can still see. k is meant to be a variable and not a constant; the tutorial tells the reader to try different k.

It would be more a disconnect to point to the graph and talk about limiting to the first 10, and it would be less interesting to see the picture with 50 when it turns out so plausibly with only 10.

We can certainly add text to emphasize that we're intentionally picking a low k.